index.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix linter errors without this kind of comment `/* eslint-disable ... */` because this type of comment is used in very few cases, and in your case, you can solve such errors yourself, and I think if implement my suggestion that this will increase your work experience and skills 😃

Reply:
TIP: If you separated the code in multiple files and you have the following issues with the linter: complaining about having multiple classes at the same file, or an unused variables, disable the linter check for that rule in the file with a /* eslint-disable rule-you-want-to-disable */

Reply:
please can you explain to me what should exactly do when linter errors are 
(max-classes-per-file) if the classes are in one file
(no-unused-vars) and (no-undef) if the files are separated in many files

Reply:
@TEAM I have seen the situation and am changing this requirement to optional. Good luck with your future endeavors 👍🏼